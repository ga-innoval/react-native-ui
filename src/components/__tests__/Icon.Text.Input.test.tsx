import React from 'react';
import { render } from '@testing-library/react-native';
import { IconTextInput } from '../Icon.Text.Input';

describe('<IconTextInput />', () => {
  let component: any = null;

  beforeEach(() => {
    component = render(
      <IconTextInput iconName="magnify" placeholder="Buscar ..." />
    );
  });

  it('should render correctly', () => {
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
