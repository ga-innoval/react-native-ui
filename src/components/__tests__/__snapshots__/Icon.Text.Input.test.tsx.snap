// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IconTextInput /> should render correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "white",
      "borderColor": "#E1E1E3",
      "borderRadius": 10,
      "borderWidth": 2,
      "flexDirection": "row",
    }
  }
>
  <View
    style={
      {
        "backgroundColor": "#E1E1E3",
        "borderBottomLeftRadius": 5,
        "borderColor": "#E1E1E3",
        "borderTopLeftRadius": 5,
        "borderWidth": 2,
        "padding": 10,
      }
    }
  >
    <Text />
  </View>
  <TextInput
    placeholder="Buscar ..."
    placeholderTextColor="#0007"
    style={
      {
        "backgroundColor": "white",
        "flex": 1,
        "marginVertical": 10,
        "paddingHorizontal": 10,
      }
    }
  />
</View>
`;
